style.css

```diff
@@ -0,0 +1,58 @@
+* {
+  margin-top: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+  text-align: center;
+}
+
+h1 {
+  font-size: 40px;
+}
+
+header h1 {
+  margin-top: 4%;
+}
+
```

Comment:
- [ ] You did a good job on the functionalities but you need to fix the below design issues.
  - Please make sure to replace the heading `Awsome Books` with `All awesome books`.
  - Please make sure the remove book button is aligned at the end of the block even if the book title and author names are very less in length.

### Your design

![image](https://user-images.githubusercontent.com/83803414/234546785-e74fb150-4dab-44b0-970a-518acca86f83.png)

### Wireframe design

![image](https://user-images.githubusercontent.com/83803414/234546862-2d5faca3-da60-45a2-94eb-2d3719dfcd15.png)